.github/workflows/CI.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: [3.7, 3.8, 3.9]
-        openmm: ["nightly", "7.5.1.", "7.6.0"]
+        openmm: ["nightly", "7.6.0"]
```

Comment:
I know this might not be an issue for the current OpenMM release (7.6), but I wonder if we want to use only `major.minor` and not `patch/build` versions, for future openmm releases. That is, only doing "7.6" instead of "7.6.0". 

Would that work as expected with mamba when these options are used? Such that doing something like `mamba install openmm=7.6` hopefully gets the latest 7.6.x for openmm.

Reply:
That is a good idea, this is how we install it:
`mamba install -y -c conda-forge openmm=${{ matrix.openmm }}`

So doing `7.6` should pull in a newer minor version, but do want to explicitly test minor versions? OpenMM doesn't follow symver so we might want to be explicit.

Reply:
Going to do 7.6 :)